### PR TITLE
Translation to spanish of challenge Computer-security in Computer-basics

### PR DESCRIPTION
**Pull request by pomarbar**

### CODE REVIEW

File: seed/challenges/04-video-challenges/computer-basics.json [hunk 1]
```diff
@@ -814,7 +814,37 @@
...
+        "Cundo se crea una clave, tú usas letras mayúsculas, minúsculas, números y símbolos, pero una de las formas más eficientes es seleccionar números y letras al azar (ejemplo: canguroSyCA67).",
+        "Por lo tanto mantente lejos de los números seguidos.",
+        "Suplantación es otra forma de ataque del cual debes estar pendiente. Son correos o páginas web que simulan sitios seguros y engañan a las personas para obtener su información personlal. Son sitios que no están conectados a donde dicen que estan.",
+        "En general hay dos formas de resguardarse de estos ataques: verificar la dirección web de la página y abrir la dirección de la página web, en otro navegador para poder comprobar si corresponde con el sitio del que dicen que procede.",
+        "Ahora hablemos sobre cifrado y HTTPS",
+        "Si te encuentras comprando en una página de internet de una tienda muy prestigiosa y necesitas dar los datos de tu tarjeta de crédito es muy razonable que sospeches que alguien puede estar viendo y copiando esta información.If you are at Starbucks trying to access your bank account, you have reason to be suspicious that someone could see and take your information.",
```
Comment (vtamara) [inline seed/challenges/04-video-challenges/computer-basics.json]: Sobra de If you are en adelante

File: seed/challenges/04-video-challenges/computer-basics.json [hunk 2]
```diff
@@ -814,7 +814,37 @@
...
+        "Es imprescindible que te fijes que la dirección de esta página web comienza con letras verdes y con HTTPS en vez de comenzar con solo HTTP.",
```
Comment (vtamara) [inline seed/challenges/04-video-challenges/computer-basics.json]: De esta pagima web -> de esta pagins web en tu navegador

File: seed/challenges/04-video-challenges/computer-basics.json [hunk 3]
```diff
@@ -814,7 +814,37 @@
...
+        "Esto significa que el sitio por el que estás enviando esta información, la envía en forma codificada",
+        "En general, asegúrese que la página sea HTTPS si necesita escribir información privada o importante.",
```
Comment (vtamara) [inline seed/challenges/04-video-challenges/computer-basics.json]: Asegurese -> asegurate

File: seed/challenges/04-video-challenges/computer-basics.json [hunk 4]
```diff
@@ -814,7 +814,37 @@
...
+        "Otras recomendaciones importantes: no uses la misma clave para diferentes cuentas importantes",
+        "Si el sitio que usas ha sido vulnerado, tu clave puede estar comprometida independientemente de su fortaleza.",
+        "Tampoco descargue archivos desconocidos.",
```
Comment (vtamara) [inline seed/challenges/04-video-challenges/computer-basics.json]: Descargue -> descargues

File: seed/challenges/04-video-challenges/computer-basics.json [hunk 5]
```diff
@@ -814,7 +814,37 @@
...
+        "Sea precavido si no reconoce archivos de tipo .pdf, .txt, .jpg.",
```
Comment (vtamara) [inline seed/challenges/04-video-challenges/computer-basics.json]: Sea -> se

File: seed/challenges/04-video-challenges/computer-basics.json [hunk 6]
```diff
@@ -814,7 +814,37 @@
...
+        "Algunos de estos archivos pueden ser muy potentes e incluso correr dentro de su computador.",
```
Comment (vtamara) [inline seed/challenges/04-video-challenges/computer-basics.json]: De su -> de tu

File: seed/challenges/04-video-challenges/computer-basics.json (hunk 5 above)
Comment (vtamara) [inline seed/challenges/04-video-challenges/computer-basics.json]: Reconoce -> reconoces

File: seed/challenges/04-video-challenges/computer-basics.json [hunk 7]
```diff
@@ -814,7 +814,37 @@
...
+        "Por último, mantenga su software actualizado, en especial el software que interactúa con internet.",
```
Comment (vtamara) [inline seed/challenges/04-video-challenges/computer-basics.json]: Mantenga su -> manten tu

File: seed/challenges/04-video-challenges/computer-basics.json [hunk 8]
```diff
@@ -814,7 +814,37 @@
...
+        "Una de las forma en que se puede comprometer su información es cuando los 'los chicos malos' encuentran huecos en las versiones viejas de los software",
```
Comment (vtamara) [inline seed/challenges/04-video-challenges/computer-basics.json]: Su -> tu

**Comment by vtamara** [comment]
Good translation, few corrections to apply.

File: seed/challenges/04-video-challenges/computer-basics.json (hunk 4 above)
Comment (Maddmam83x) [inline seed/challenges/04-video-challenges/computer-basics.json]: https://api.github.com

**Comment by ltegman** [comment]
Thanks both of you! :+1:

**Comment by Glenn230593** [comment]
2016-03-04 22:03 GMT+01:00, Logan Tegman <notifications@github.com>:
> Merged #7328.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/FreeCodeCamp/FreeCodeCamp/pull/7328#event-578365281


-- 
Glenntobby@gmail

**Pull request merged**